src/main/webapp/app/shared/components/checklist-check/checklist-check.component.html

```diff
@@ -1,7 +1,6 @@
 @if (checkAttribute) {
     <fa-icon [icon]="faCheckCircle" [size]="size" class="checked" [style]="{ color: iconColor }" />
-}
-@if (!checkAttribute) {
-    <fa-icon [icon]="faTimes" [size]="size" class="unchecked" [style]="{ fontSize: '1.5em', color: iconColor }" />
+} @else {
+    <fa-icon [icon]="faTimes" [size]="size" class="unchecked" [style]="{ color: iconColor }" />
```

Comment:
I didn't notice earlier, but the unchecked class is a bit confusing. Could we use a default icon color instead?

For the font size, you should be able to use either fs-5 or fs-4. (I'm not entirely sure which is the correct value.)